src/test/java/com.gigssandbox/UserCommandHandlerTest.java

```diff
@@ -104,7 +104,7 @@ public void shouldReturnNotRegisteredResultWhenLastCommandWasLogInAndUserWasNotR
     @Test
     public void shouldReturnIncorrectPasswordResultWhanLastCommandWasLogInAndUserEnteredWrongPassword() {
         String incorrectPassword = "in black";
-        users.put(username, user);
+        userCommandHandler.process(new Command(CommandType.REGISTER, List.of(username, password)));
```

Comment:
can we use CommandFactory here right now? I think this will make test code more clear. 

```
commandFactory.create(String commandAsString);   
```

And we can make Command package-private now to let only CommandFactory create it